src/test/perl/test-cafapplication.t

```diff
@@ -61,6 +61,16 @@ is($newapp->option($OPTION_CFGFILE), $cfgfile,
 is($newapp->option('myoption'), $value,
    "myoption value from configfile");
 
+# 1st format short -cfgile path/tofile
```

Comment:
this is the intention of the PR to also support `-cfgfile` besides `--cfgfile`, and this is the test for `-cfgfile=abc`